tests/suite_test.go

```diff
@@ -167,12 +170,12 @@ func (suite *IntegrationTestSuite) SetupSuite() {
 	if !skipDeploy {
 		loggr.Info("Deploying DSPA...")
 		err = testUtil.DeployDSPA(suite.T(), ctx, clientmgr.k8sClient, DSPA, DSPANamespace, DeployTimeout, PollInterval)
-		assert.NoError(suite.T(), err)
+		require.NoError(suite.T(), err)
 		loggr.Info("Waiting for DSPA pods to ready...")
 	}
 
 	err = testUtil.WaitForDSPAReady(suite.T(), ctx, clientmgr.k8sClient, DSPA.Name, DSPANamespace, DeployTimeout, PollInterval)
-	assert.NoError(suite.T(), err)
+	require.NoError(suite.T(), err, fmt.Sprintf("Error Deploying DSPA:\n%s", printConditions(ctx, DSPA, DSPANamespace, clientmgr.k8sClient)))
```

Comment:
this isn't related to your PR. extract to a new PR.

Reply:
still applies

Reply:
^bump i suppose?